dallinger/frontend/static/scripts/tracking/load-tracker.js

```diff
@@ -0,0 +1,65 @@
+/*global require */
+/*jshint esversion: 6 */
+
+var dlgr = window.dlgr = (window.dlgr || {});
+
+if (window.getUrlParameter === undefined) {
+  var getUrlParameter = function getUrlParameter(sParam) {
```

Comment:
I think this is included in the new `dallinger2.js`. @davisagli could confirm.

Reply:
Even if it is, we need to have this function available in `tracker.js` to load on sites which don't have `dallinger2.js` loaded. I guess we should look for the method on `window.dlgr` instead of `window` though, and consider using a shared library for these methods (see below).

Reply:
Gotcha, okay.